applicatie/main/verwerking.py

```diff
@@ -28,16 +28,11 @@ def __init__(self, jsonbestand):
                 # json bestand inlezen
                 self.data_bestand, self.fouten = laad_json_bestand(jsonbestand)
 
-
-
-
-                # TODO UNCOMMENT ONDERSTAANDE 2 REGELS ZODRA JSON ENCODING ISSUES VOOR BESTANDEN ZIJN OPGELOST:
                 # if not self.fouten:
                 #     self.fouten = geef_fouten(jsonbestand.filename)
-                self.fouten = geef_fouten(jsonbestand.filename) # OVERSCHRIJFT FOUTEN UIT laad_json_bestand(), HAAL DIT WEG.
-
-
-
+                self.fouten = geef_fouten(jsonbestand.filename) # TODO OVERSCHRIJFT self.fouten uit laad_json_bestand(),
+                # dit omdat ik geen vat kreeg op de output van laad_json_bestand(), misschien kan refactoring voor een goede
+                # exception handling zorgen van laad_json_bestand().
```

Comment:
Hulp nodig? Ik zou de fouten eerder aanvullen dan overschrijven.

Reply:
ja mee eens, alleen dan werkt het programma dus niet, want ik heb geen goede JSON bestanden zonder fouten die door het programma heen komen... elke "goede" json file die Theo me stuurt bevat fouten, en wordt door onze Ocido code al afgevangen dat het geen goed json bestand betreft. Er zit dus eigenlijk een bug in de code, daarom wilde ik wachten tot Theo goede testbestanden opgestuurd heeft, maar misschien is het beter om het nu te laten falen en daar dan een bug issue voor aan te maken?

Reply:
Ik zal ze inderdaad aanvullen voor nu, met dus de kanttekening dat het dan niet meer mogelijk is om naar het eigenlijke programma te komen, omdat elk bestand fouten heeft.